From a1a0f3cf9b80f1d13f7dc10cf540c1ca19570b70 Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Thu, 16 Jan 2014 09:31:59 -0500
Subject: [PATCH 09/14] switch rhel7 machine types to big bios.

Message-id: <1389864720-14128-3-git-send-email-kraxel@redhat.com>
Patchwork-id: 56737
O-Subject: [RHEL-7 qemu-kvm PATCH 2/3] switch rhel7 machine types to big bios.
Bugzilla: 1038603
RH-Acked-by: Orit Wasserman <owasserm@redhat.com>
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>

related upstream commit: bcf2b7d2af7c54bb42be1229df9e78ba7d08d2a7

Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
---
 hw/i386/pc_piix.c             | 1 +
 hw/i386/pc_q35.c              | 1 +
 redhat/qemu-kvm.spec.template | 6 ++++--
 3 files changed, 6 insertions(+), 2 deletions(-)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 hw/i386/pc_piix.c             | 1 +
 hw/i386/pc_q35.c              | 1 +
 redhat/qemu-kvm.spec.template | 6 ++++--
 3 files changed, 6 insertions(+), 2 deletions(-)

diff --git a/hw/i386/pc_piix.c b/hw/i386/pc_piix.c
index 29c013a..e061877 100644
--- a/hw/i386/pc_piix.c
+++ b/hw/i386/pc_piix.c
@@ -756,6 +756,7 @@ static QEMUMachine pc_machine_rhel700 = {
     .hot_add_cpu = pc_hot_add_cpu,
     .max_cpus = 255,
     .is_default = 1,
+    .default_machine_opts = "firmware=bios-256k.bin",
     DEFAULT_MACHINE_OPTIONS,
 };
 
diff --git a/hw/i386/pc_q35.c b/hw/i386/pc_q35.c
index 86c0c90..8d1a714 100644
--- a/hw/i386/pc_q35.c
+++ b/hw/i386/pc_q35.c
@@ -280,6 +280,7 @@ static QEMUMachine pc_q35_machine_rhel700 = {
     .init = pc_q35_init_rhel700,
     .hot_add_cpu = pc_hot_add_cpu,
     .max_cpus = 255,
+    .default_machine_opts = "firmware=bios-256k.bin",
     DEFAULT_MACHINE_OPTIONS,
 };
 
